.github/workflows/main-e2e.yml

```diff
@@ -267,26 +163,14 @@ jobs:
 
   roles-tests:
     runs-on: ubuntu-latest
-    needs: [pre-reqs, create-integrations]
+    needs: [pre-reqs]
```

Comment:
do we not need to create the integrations anymore?

Reply:
No, I removed that job entirely. The way it was setup, that test would run 4 files which created the integrations required for the tests afterwards, basically by uploading a temporary file with ids in it. This was very broken and difficult to debug, leading to the flaky failures. I rewrote that process so that single test files would do their own setup/teardown, which makes it much simpler to debug.

That is why there are so many deletions, before it was like run 4 files, one to create an integration, then one to validate it, one to update, and one to revalidate. Collapsing them into 1 file allows us to keep the id reference in the test and keep the files idempotent

Reply:
I think it was done that way because the terraform process took so long. Now that it is fast though doing the setup in the files seems like the way to go